content/300-guides/050-database/880-troubleshooting-orm/100-help-articles/425-nextjs-prisma-client-monorepo.mdx

```diff
@@ -0,0 +1,68 @@
+---
+title: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaTitle: 'Using Prisma Client in a Next.js project in a monorepo setup'
+metaDescription: 'Using Prisma Client in a Next.js project in a monorepo setup'
+---
+
+## Problem
+
+If you use Prisma Client in a [Next.js](https://nextjs.org/) application within a monorepo, you will likely run into an error such as the following:
+
+```
+Error: ENOENT: no such file or directory, open '/prisma-next-mono-reproduction/packages/service/.next/cache/webpack/client-development/schema.prisma'
+```
+
+Assume you have a monorepo with the following structure:
+
+```
+.
+├── packages/
+│   ├── prisma/
+│   │   ├── client/ <-- Custom output location for the generated Prisma Client
+│   │   └── schema.prisma
+│   └── service/
+│       ├── pages/
+│       │   └── api/
+│       │       └── test.js
+│       ├── next.config.js
+│       └── package.json
+├── pnpm-workspaces.yaml
+├── package.json
+└── vercel.json
+```
+
+This error mentioned above occurs as a result of a bundling problem during Next.js's bundling process. The `schema.prisma` file is treated as an artifact that should be bundled along with the rest of the `/api/test` endpoint's bundle.
+
+This causes an issue as Prisma Client expects `schema.prisma` to be available at a specific location in relation to the bundled code.
```

Comment:
What is the problem or bug here? This does not explain that the file should be somewhere, but Next does not actually put it there. The file is not copied/included at all actually if I understood correctly.